### ISSUE TITLE
Questions in the vessel_loss_2dv1_sqmax

### ISSUE THREAD

**Issue by Puzzled-Hui** [batmanlab/vessel-flow-consistency-ssl#3]
Thanks for your impressive work!

I am curious about the parameter multiplier in the loss function _vessel_loss_2dv1_sqmax_, as original code belows 

` Consistency loss
if l_consistency:
            v1x = resample_from_flow_2d(v1+0, v1+0)
            loss = loss - l_consistency * F.cosine_similarity(v1, v1x).mean()
       Check for cosine similarity
        if l_cosine:
            v1x = resample_from_flow_2d(v1+0, v1+0)
            sim = F.cosine_similarity(v1, v1x)
            sim0 = sim*0
            loss = loss - l_consistency * torch.min(sim, sim0).mean()`

**Why in the l_cosine setting, the loss is the loss - l_consistency * torch.min(sim, sim0).mean() rather than the loss - l_cosine* torch.min(sim, sim0).mean()**

**Comment by rohitrango** [comment]
That is correct, the multiplier should be `l_cosine`.
It was a hacky code on my part because I didn't want to change a huge number of config files (to weigh consistency and cosine equally).

I would appreciate if you can submit a PR.
